<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtShanoirSearchWidget</class>
 <widget class="QWidget" name="QtShanoirSearchWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>439</width>
    <height>66</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>421</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="3">
     <widget class="QLineEdit" name="datasetNameLineEdit">
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="examinationDateLabel">
      <property name="text">
       <string>Examination date:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QDateEdit" name="examinationDateDateEdit"/>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="studyNameLineEdit">
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="patientNameLabel">
      <property name="text">
       <string>Patient name:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="datasetNameLabel">
      <property name="text">
       <string>Dataset name:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="studyNameLabel">
      <property name="text">
       <string>Study name:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLineEdit" name="patientNameLineEdit">
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
